### PR TITLE
Auth 345 - Update APS helm chart to 0.1.5

### PR DESCRIPTION
**Pull request by rpopa**
This has https://github.com/Alfresco/alfresco-dbp-deployment/pull/114 merged into this branch in order to make it work

### CODE REVIEW

File: charts/incubator/alfresco-dbp/values.yaml [hunk 1]
```diff
@@ -185,7 +200,7 @@ alfresco-content-services:
         -Dalfresco.cluster.enabled=true
         -Ddeployment.method=HELM_CHART
         -Xms2000M -Xmx2000M
-        -Ddsync.service.uris=\"$SYNC_SERVICE_URI\"
+        -Dsync.service.uris=\"$SYNC_SERVICE_URI\"
```
Comment (rgauss) [inline charts/incubator/alfresco-dbp/values.yaml]: Are we sure this has changed?

Comment (rpopa) [reply charts/incubator/alfresco-dbp/values.yaml]: Reverted

File: charts/incubator/alfresco-dbp/requirements.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 dependencies:
   - name: alfresco-content-services
-    version: 3.0.0
+    version: 2.1.0
     condition: alfresco-content-services.enabled
     repository: https://kubernetes-charts.alfresco.com/incubator
   - name: alfresco-infrastructure
     version: 5.1.1
```
Comment (rgauss) [inline charts/incubator/alfresco-dbp/requirements.yaml]: Per team discussion this should be downgraded to `4.1.1` on master branch.

File: charts/incubator/alfresco-dbp/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: alfresco-dbp
-version: 0.8.4
+version: 0.8.4-andreif
```
Comment (jamalkm) [inline charts/incubator/alfresco-dbp/Chart.yaml]: The version should be `1.5` as per this [comment](https://issues.alfresco.com/jira/browse/AUTH-345?focusedCommentId=632370&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-632370).

**Comment by rgauss** [comment]
> Can you also remove the Sync Service values from here and here?

@cristina-sirbu, don't we still need to set those based on the URL the DBP is being deployed to?  See [comment on previous PR](https://github.com/Alfresco/alfresco-dbp-deployment/pull/114#discussion_r313865075).

**Comment by rgauss** [comment]
@flv22, don't we still need to set the URLs for APW that were removed in commit c96a828?

**Comment by cristina-sirbu** [comment]
@rgauss As far as I saw in the ACS chart they set that property to be **externalHost** (https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/templates/config-repository.yaml#L69) So I don't think we need to override it explicitly anymore.

**Pull request merged**